interop.md

```diff
@@ -21,7 +21,7 @@ For example, Mockito's `when` function requires backticks when used from Kotlin:
 
 ```kotlin
 val callable = Mockito.mock(Callable::class.java)
-Mockito.`when`(callable.call()).thenReturn(/* … */)
+Mockito.when(callable.call()).thenReturn(/* … */)
```

Comment:
`when` is a hard keyword in Kotlin. Backticks are required here. :)

Reply:
thank you. will revert it.

Reply:
@JakeWharton I'm no longer able to edit this PR, can you please just rever this change? The remaining changes are OK.